requirements.txt

```diff
@@ -1,3 +1,3 @@
-steamship==2.17.22
 openai==0.27.8
-tenacity==8.2.2
\ No newline at end of file
+tenacity==8.2.2
+steamship @ git+https://github.com/steamship-core/python-client@streaming-generators
```

Comment:
probably need to remove this first

Reply:
Yup!  This can't roll out until the stuff on staging makes it to prod and these things get merged in the python sdk.